tests/server/test_params.py

```diff
@@ -0,0 +1,226 @@
+"""Unit tests for granular sensor authentication in api.php."""
```

Comment:
```suggestion
"""Unit tests for parameter parsing."""
```
? bc I'm pretty sure this doesn't deal with auth